public/gameInfo.php

```diff
@@ -666,78 +671,74 @@ function ResetProgress() {
             if (isset($user) && ($permissions >= Permissions::Developer || ($isFullyFeaturedGame && $permissions >= Permissions::JuniorDeveloper))) {
                 echo "<div class='devbox mb-3'>";
                 echo "<span onclick=\"$('#devboxcontent').toggle(); return false;\">Dev ▼</span>";
-                echo "<div id='devboxcontent' style='display: none'>";
-
-                // Only allow developers to rename a game
-                if ($permissions >= Permissions::Developer) {
-                    echo "<div class='mb-3'><a href='/attemptrename.php?g=$gameID'>Rename Game</a></div>";
-                }
+                // echo "<div id='devboxcontent' style='display: none'>";
```

Comment:
this `display:none` is required to have the devbox start in a collapsed state. I'm assuming it was commented out for testing and should be restored while removing the next line.

Reply:
Whoops, that wasn't supposed to be in there indeed. Got confused with my stashes there.